src/components/Main.js

```diff
@@ -1,18 +1,18 @@
-import React from 'react';
-import { Switch, Route } from 'react-router-dom'
+import React from "react";
+import { Switch, Route } from "react-router-dom";
 
-import Movies from './Movies';
-import AddMovie from './AddMovie';
-import SingleMovie from './SingleMovie';
+import Movies from "./Movies";
+import AddMovie from "./AddMovie";
+import SingleMovie from "./SingleMovie";
 
 const Main = () => (
   <main>
     <Switch>
-      <Route exact path='/' component={Movies}/>
-      <Route path='/add' component={AddMovie}/>
-      <Route path='/singlemovie/:kica' component={SingleMovie}/>
+      <Route exact path="/" component={Movies} />
+      <Route path="/add" component={AddMovie} />
+      <Route path="/singlemovie/:kica" component={SingleMovie} />
```

Comment:
"/singlemovie/:kica" kakva je ovo sprdnja, za ne bi trebalo id? :)

Reply:
To je jos od kad je Kica kucao na casu, napisao iz zezanja da nam pokaze da moze bilo sta da stoji i ostalo, nisam primetio :D